recipe/meta.yaml

```diff
@@ -8,43 +8,45 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4e913d807aa6598c320e8a50c50e2ee0602bc45240b485e3f8bc06f13060084c
+  sha256: 21f535de2e7a82c94c130c054adb5c7421d480d5619d61073996e2f89bcb879b
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<39]
 
 requirements:
   host:
     - python
     - pip
     - pytest-runner
```

Comment:
I think pytest-runner is not needed, see https://github.com/wolph/portalocker/blob/v3.0.0/pyproject.toml#L3 and https://github.com/wolph/portalocker/commit/05331bfa0aeb06b29eb7a28510466de92c438d5e

Reply:
removed